docs/edge.md

```diff
@@ -1,92 +1,141 @@
 
-Average loss = (Sum of losses) / (Number of losing trades)
+$$ \text{average_profit} = \mu_{win} = \frac{\text{sum_of_profits}}{\text{count_winning_trades}} = \frac{\sum^{o \in T_{win}} o}{|T_{win}|} $$
+
+Similarly, we can calculate the average loss, $\mu_{lose}$, as follows:
+
+$$ \text{average_loss} = \mu_{lose} = \frac{\text{sum_of_losses}}{\text{count_losing_trades}} = \frac{\sum^{o \in T_{lose}} o}{|T_{lose}|} $$
+
+Finally, we can calculate the Risk Reward ratio as follows:
+
+$$ R = \frac{\text{average_profit}}{\text{average_loss}} = \frac{\mu_{win}}{\mu_{lose}}\\ $$
 
-R = (Average profit) / (Average loss)
-```
+
+??? Example "Worked example of $R$ calculation using mean profit/loss"
+    Let's say the strategy that we are using makes an average win $\mu_{win} = 2.06$ and an average loss $\mu_{loss} = 4.11$.<br>
+    We calculate the risk reward ratio as follows:<br>
+    $R = \frac{\mu_{win}}{\mu_{loss}} = \frac{2.06}{4.11} = 0.5012...$
+    
 
 ### Expectancy
 
-At this point we can combine *W* and *R* to create an expectancy ratio. This is a simple process of multiplying the risk reward ratio by the percentage of winning trades and subtracting the percentage of losing trades, which is calculated as follows:
+By combining the Win Rate $W$ and and the Risk Reward ratio $R$ to create an expectancy ratio $E$. A expectance ratio is the expected return of the investment made in a trade. We can compute the value of $E$ as follows:
 
-```
-Expectancy Ratio = (Risk Reward Ratio X Win Rate) – Loss Rate = (R X W) – L
-```
+$$E = R * W - L$$
 
-So lets say your Win rate is 28% and your Risk Reward Ratio is 5:
+!!! Example "Calculating $E$"
+    Let's say that a strategy has a win rate $W = 0.28$ and a risk reward ratio $R = 5$. What this means is that the strategy is expected to make 5 times the investment around on 28% of the trades it makes. Working out the example:<br>
+    $E = R * W - L = 5 * 0.28 - 0.72 = 0.68$
+    <br>
 
-```
-Expectancy = (5 X 0.28) – 0.72 = 0.68
-```
+The expectancy worked out in the example above means that, on average, this strategy' trades will return 1.68 times the size of its losses. Said another way, the strategy makes $1.68 for every $1 it loses, on average. 
 
-Superficially, this means that on average you expect this strategy’s trades to return 1.68 times the size of your loses. Said another way, you can expect to win $1.68 for every $1 you lose. This is important for two reasons: First, it may seem obvious, but you know right away that you have a positive return. Second, you now have a number you can compare to other candidate systems to make decisions about which ones you employ.
+You canThis is important for two reasons: First, it may seem obvious, but you know right away that you have a positive return. Second, you now have a number you can compare to other candidate systems to make decisions about which ones you employ.
```

Comment:
```suggestion
This is important for two reasons: First, it may seem obvious, but you know right away that you have a positive return. Second, you now have a number you can compare to other candidate systems to make decisions about which ones you employ.
```